pkg/kv/kvpb/errors.go

```diff
@@ -305,6 +307,50 @@ const (
 	NumErrors int = 45
 )
 
+// Register the migration of all errors that used to be in the roachpb package
+// and are now in the kv/kvpb package.
+func init() {
+	roachpbPath := reflect.TypeOf(roachpb.Key("")).PkgPath()
+	// TODO: this doesn't have the corresponding ErrorDetailType.
```

Comment:
What is this comment supposed to mean?

Reply:
The comment on `ErrorDetailInterface` interface seems to imply that all protobuf-backed errors need to implement it, and currently `UnhandledRetryableError` and `UnhandledRetryableError` don't. Not sure how important this fact is, but I don't think it changes anything about this PR, just something to think about. I removed the TODOs though.